README.md

```diff
@@ -121,6 +121,35 @@ $ export OMP_NUM_THREADS=4
 
 Before calling the `tdms` executable.
 
+## Citation
+
+If you used TDMS in your research and found it helpful, please cite this work.
+<!-- [zenodo/FIXME](https://zenodo.org/) -->
+
+<!-- If you use TDMS in your work and have examples that you would like to share with other users, please get in touch with us at -->
+<!-- [contact_address)[mailto:FIXME] -->
+<details>
+<summary>BibTEX</summary>
+
+```bibtex
+@misc{tdms,
+  author       = Munro, Peter and others,
+  title        = {TDMS - The Time-Domain Maxwell Solver},
+  url          = {https://github.com/UCL/TDMS}
+}
+```
```

Comment:
This is the bibtex entry created by github for the given citation.cff file:
```bibtex
@software{Munro_TDMS_-_Time,
author = {Munro, Peter and Cunliffe, Sam and Giordano, Mosè and Graham, William and Kriezis, Emmanouil and Török, Peter and Young, Tom},
license = {GPL-3.0},
title = {{TDMS - Time Domain Maxwell Solver}},
url = {https://github.com/UCL/TDMS}
}
```
which I think also correctly preserves cases in the title value.

Reply:
Thanks!  🦅 👀